security/filters.py

```diff
@@ -0,0 +1,62 @@
+from django import forms
+from django.db.models import Q
+from django.utils.translation import ugettext_lazy as _
+
+from itertools import chain
+
+from pyston.filters.default_filters import SimpleMethodEqualFilter
+from pyston.filters.exceptions import FilterValueError
+
+from is_core.filters import UIFilterMixin
+
+from .enums import CeleryTaskLogState, CeleryTaskRunLogState
+from .models import CeleryTaskRunLog
+
+
+class CeleryTaskLogStateFilter(UIFilterMixin, SimpleMethodEqualFilter):
+
+    EMPTY_CHOICE = ('', _('--------'))
```

Comment:
I think that those dashes don't need to be translated.

Reply:
true story

Reply:
It is not resolved :D I am still seeing it :-)

Reply:
I now